Index: firstboot-qt/data/firstboot.service
===================================================================
--- firstboot-qt.orig/data/firstboot.service
+++ firstboot-qt/data/firstboot.service
@@ -1,6 +1,6 @@
 [Unit]
 Description=first config system
-After=systemd-user-sessions.service 
+After=systemd-user-sessions.service plymouth-quit.service
 Before=display-manager.service 
 
 [Service]
Index: firstboot-qt/data/run-firstboot
===================================================================
--- firstboot-qt.orig/data/run-firstboot
+++ firstboot-qt/data/run-firstboot
@@ -15,9 +15,13 @@
 fi
 
 cat > /tmp/.firstboot-script <<EOF
-kwin_x11&
-exec /bin/firstboot-qt
+kwin_x11 &
+#exec /bin/firstboot-qt
+/bin/firstboot-qt
+/bin/plymouth reactivate
 EOF
 
-startx /tmp/.firstboot-script --/usr/bin/X :0
-
+#/usr/bin/plymouth quit --retain-splash
+#/usr/bin/plymouth deactivate
+/bin/plymouth quit
+xinit /tmp/.firstboot-script -- /usr/bin/X :0
