diff -Nur kamera-15.04.3/kioslave/kamera.cpp kamera-15.04.3n/kioslave/kamera.cpp
--- kamera-15.04.3/kioslave/kamera.cpp	2015-07-23 19:15:08.033820253 +0000
+++ kamera-15.04.3n/kioslave/kamera.cpp	2015-07-23 19:15:18.979820057 +0000
@@ -657,7 +657,7 @@
 			entry.insert(KIO::UDSEntry::UDS_NAME,path_quote(xname));
 			// do not confuse regular users with the @usb... 
 			entry.insert(KIO::UDSEntry::UDS_DISPLAY_NAME,*it);
-			listEntry(entry, false);
+			listEntry(entry);
 		}
 
 		QMap<QString,QString>::iterator portsit;
@@ -670,9 +670,8 @@
 			entry.insert(KIO::UDSEntry::UDS_NAME, path_quote(portsit.value()+"@"+portsit.key()));
 
 			entry.insert(KIO::UDSEntry::UDS_ACCESS,(S_IRUSR | S_IRGRP | S_IROTH |S_IWUSR | S_IWGRP | S_IWOTH));
-			listEntry(entry, false);
+			listEntry(entry);
 		}
-		listEntry(entry, true);
 
 		finished();
 		return;
@@ -732,7 +731,7 @@
 	for(int i = 0; i < gp_list_count(dirList); ++i) {
 		gp_list_get_name(dirList, i, &name);
 		translateDirectoryToUDS(entry, QString::fromLocal8Bit(name));
-		listEntry(entry, false);
+		listEntry(entry);
 	}
 
 	CameraFileInfo info;
@@ -742,21 +741,21 @@
 		// we want to know more info about files (size, type...)
 		gp_camera_file_get_info(m_camera, tocstr(directory), name, &info, m_context);
 		translateFileToUDS(entry, info, QString::fromLocal8Bit(name));
-		listEntry(entry, false);
+		listEntry(entry);
 	}
 	if (!directory.compare("/")) {
 		CameraText text;
 		if (GP_OK == gp_camera_get_manual(m_camera, &text, m_context)) {
 			translateTextToUDS(entry, "manual.txt", text.text);
-			listEntry(entry, false);
+			listEntry(entry);
 		}
 		if (GP_OK == gp_camera_get_about(m_camera, &text, m_context)) {
 			translateTextToUDS(entry, "about.txt", text.text);
-			listEntry(entry, false);
+			listEntry(entry);
 		}
 		if (GP_OK == gp_camera_get_summary(m_camera, &text, m_context)) {
 			translateTextToUDS(entry, "summary.txt", text.text);
-			listEntry(entry, false);
+			listEntry(entry);
 		}
 	}
 
@@ -765,7 +764,6 @@
 	gp_list_free(dirList);
 	gp_list_free(specialList);
 
-	listEntry(entry, true); // 'entry' is not used in this case - we only signal list completion
 	finished();
 }
 
