diff -Nur chromium-45.0.2454.99/chrome/browser/password_manager/native_backend_kwallet_x.cc chromium-45.0.2454.99n/chrome/browser/password_manager/native_backend_kwallet_x.cc
--- chromium-45.0.2454.99/chrome/browser/password_manager/native_backend_kwallet_x.cc	2015-09-22 04:24:11.000000000 +0800
+++ chromium-45.0.2454.99n/chrome/browser/password_manager/native_backend_kwallet_x.cc	2015-09-25 15:38:18.694332236 +0800
@@ -38,10 +38,10 @@
 const char kKWalletFolder[] = "Chrome Form Data";
 
 // DBus service, path, and interface names for klauncher and kwalletd.
-const char kKWalletServiceName[] = "org.kde.kwalletd";
-const char kKWalletPath[] = "/modules/kwalletd";
+const char kKWalletServiceName[] = "org.kde.kwalletd5";
+const char kKWalletPath[] = "/modules/kwalletd5";
 const char kKWalletInterface[] = "org.kde.KWallet";
-const char kKLauncherServiceName[] = "org.kde.klauncher";
+const char kKLauncherServiceName[] = "org.kde.klauncher5";
 const char kKLauncherPath[] = "/KLauncher";
 const char kKLauncherInterface[] = "org.kde.KLauncher";
 
@@ -368,7 +368,7 @@
                                "start_service_by_desktop_name");
   dbus::MessageWriter builder(&method_call);
   std::vector<std::string> empty;
-  builder.AppendString("kwalletd");     // serviceName
+  builder.AppendString("kwalletd5");     // serviceName
   builder.AppendArrayOfStrings(empty);  // urls
   builder.AppendArrayOfStrings(empty);  // envs
   builder.AppendString(std::string());  // startup_id
--- chromium-47.0.2526.80/chrome/browser/ssl/bad_clock_blocking_page.cc	2015-12-09 04:02:51.000000000 +0800
+++ chromium-47.0.2526.80n/chrome/browser/ssl/bad_clock_blocking_page.cc	2015-12-13 14:46:06.925645923 +0800
@@ -109,9 +109,9 @@
       {"/usr/local/bin/gnome-control-center", "datetime"},
       {"/opt/bin/gnome-control-center", "datetime"},
       // KDE
-      {"/usr/bin/kcmshell4", "clock"},
-      {"/usr/local/bin/kcmshell4", "clock"},
-      {"/opt/bin/kcmshell4", "clock"},
+      {"/usr/bin/kcmshell5", "clock"},
+      {"/usr/local/bin/kcmshell5", "clock"},
+      {"/opt/bin/kcmshell5", "clock"},
   };
 
   base::CommandLine command(base::FilePath(""));
